bagh/bagh.json

```diff
@@ -5,6 +5,14 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "versioning": {
```

Comment:
Het is nu op het niveau van de dataset gedefinieerd. Zou het nog kunnen voorkomen dat dit per tabel binnen de dataset weer kan verschillen? Alhoewel ik wel voor voorkeur zou hebben voor het dataset niveau.

Reply:
Nee, kan niet, omdat alle referenties binnen deze schema zijn versie/datum afhankelijk zijn.